dev/AppLifecycle/FileActivatedEventArgs.h

```diff
@@ -44,11 +44,11 @@ namespace winrt::Microsoft::Windows::AppLifecycle::implementation
             }
         }
 
-        static IActivatedEventArgs Deserialize(winrt::Windows::Foundation::Uri const& uri)
+        static winrt::Windows::Foundation::IInspectable Deserialize(winrt::Windows::Foundation::Uri const& uri)
         {
             auto query = uri.QueryParsed();
-            std::wstring verb = query.GetFirstValueByName(L"Verb").c_str();
-            std::wstring file = query.GetFirstValueByName(L"File").c_str();
+            auto verb = std::wstring(query.GetFirstValueByName(L"Verb"));
+            auto file = std::wstring(query.GetFirstValueByName(L"File"));
```

Comment:
How trustworthy is the URI? If the URI doesn't have a `Verb` or doesn't have a `File`, the `GetFirstValueByName` will throw an exception and the app will probably just crash out. Is that okay?

Reply:
Adding this comment to this open issue in AppLifeycle: #926